hail/src/main/scala/is/hail/variant/MatrixTable.scala

```diff
@@ -498,7 +498,7 @@ class MatrixTable(val hc: HailContext, val ast: MatrixIR) {
 
   def annotateGlobal(a: Annotation, t: Type, name: String): MatrixTable = {
     new MatrixTable(hc, MatrixMapGlobals(ast,
-      ir.InsertFields(ir.Ref("global", ast.typ.globalType), FastSeq(name -> ir.Literal(t, a, ir.genUID())))))
+      ir.InsertFields(ir.Ref("global", ast.typ.globalType), FastSeq(name -> ir.Literal(t, a)))))
```

Comment:
I think you probably want this to be `Literal.coerce`? `PruneSuite` is failing because it's passing emittable types into this function.

Reply:
yes, good catch! I don't know why I just saw the failure now.